src/OpenSage.Game/Gui/Apt/ActionScript/Function.cs

```diff
@@ -1,13 +1,36 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using OpenSage.Gui.Apt.ActionScript.Opcodes;
 
 namespace OpenSage.Gui.Apt.ActionScript
 {
+    [Flags]
+    public enum FunctionPreloadFlags
+    {
+        PreloadExtern = 0x010000,   //this seems to be added by EA
```

Comment:
It's not critical, but I usually like enum values to be in ascending numeric order, unless there's a good reason not to.

Reply:
I did that because that's the order of bits. Otherwise i always get confused